mkosi/manifest.py

```diff
@@ -214,6 +216,17 @@ def record_pkg_packages(self) -> None:
                 self.source_packages[source] = source_package
             source_package.add(package)
 
+    def record_extension_release(self) -> None:
+        if self.context.config.output_format not in (OutputFormat.sysext, OutputFormat.confext):
+            return
+
+        with complete_step("Recording sysext / confext information in manifest…"):
+            type_ = "sysext" if self.context.config.output_format == OutputFormat.sysext else "confext"
```

Comment:
No need to use `type_`, you can just use `type`, it doesn't matter that it overrides a builtin.

Reply:
Very ugly. But done.